main/parse.c

```diff
@@ -1635,7 +1635,7 @@ static boolean enableLanguageKind (
 }
 
 static boolean enableLanguageKindLong (
-	const langType language, const const char *kindLong, const boolean mode)
+	const langType language, const char *kindLong, const boolean mode)
```

Comment:
Yes. @tmm1, can I ask you to change the line as @b4n said?
Then I will merge your changes.